extensions/void/package.json

```diff
@@ -15,35 +15,274 @@
+            "command-r-plus:104b",
+            "deepseek-coder-v2",
+            "deepseek-coder-v2:16b",
+            "deepseek-coder-v2:236b",
+            "falcon2",
+            "falcon2:11b",
+            "firefunction-v2",
+            "firefunction-v2:70b",
+            "gemma",
+            "gemma:2b",
+            "gemma:7b",
+            "gemma2",
+            "gemma2:2b",
+            "gemma2:9b",
+            "gemma2:27b",
+            "llama2",
+            "llama2:7b",
+            "llama2:13b",
+            "llama2:70b",
+            "llama3",
+            "llama3:8b",
+            "llama3:70b",
+            "llama3-chatqa",
+            "llama3-chatqa:8b",
+            "llama3-chatqa:70b",
+            "llama3-gradient",
+            "llama3-gradient:8b",
+            "llama3-gradient:70b",
+            "llama3.1",
+            "llama3.1:8b",
+            "llama3.1:70b",
+            "llama3.1:405b",
+            "llava",
+            "llava:7b",
+            "llava:13b",
+            "llava:34b",
+            "llava-llama3",
+            "llava-llama3:8b",
+            "llava-phi3",
+            "llava-phi3:3.8b",
+            "mistral",
+            "mistral:7b",
+            "mistral-large",
+            "mistral-large:123b",
+            "mistral-nemo",
+            "mistral-nemo:12b",
+            "mixtral",
+            "mixtral:8x7b",
+            "mixtral:8x22b",
+            "moondream",
+            "moondream:1.8b",
+            "openhermes",
+            "openhermes:v2.5",
+            "phi3",
+            "phi3:3.8b",
+            "phi3:14b",
+            "phi3.5",
+            "phi3.5:3.8b",
+            "qwen",
+            "qwen:7b",
+            "qwen:14b",
+            "qwen:32b",
+            "qwen:72b",
+            "qwen:110b",
+            "qwen2",
+            "qwen2:0.5b",
+            "qwen2:1.5b",
+            "qwen2:7b",
+            "qwen2:72b",
+            "smollm",
+            "smollm:135m",
+            "smollm:360m",
+            "smollm:1.7b"
+          ]
+        },
+        "void.ollama.providerSettings": {
+          "type": "object",
+          "properties": {
+            "baseURL": {
+              "type": "string",
+              "default": "http://127.0.0.1:11434",
+              "description": "Ollama base URL. Local API server can be started with `OLLAMA_ORIGINS=* ollama serve`"
+            },
+            "headers": {
+              "type": "object",
+              "description": "Custom headers to include in the requests."
+            }
+          }
+        },
+        "void.greptile.apiKey": {
+          "type": "string",
+          "description": "Greptile API key."
+        },
+        "void.greptile.providerSettings": {
+          "type": "object",
+          "properties": {
+            "headers": {
+              "type": "object",
+              "description": "Custom headers to include in the requests.",
+              "properties": {
+                "X-Github-Token": {
```

Comment:
Should we change this to `void.greptile.githubToken`?

Reply:
I think so, yes.